libs/src/sanityChecks/index.js

```diff
@@ -15,6 +17,8 @@ class SanityChecks {
    */
   async run (creatorNodeWhitelist = null) {
     await isCreator(this.libs)
+    await sanitizeNodes(this.libs)
```

Comment:
shouldn't we retry some x number of times in case of failure?

Reply:
I would say that since this runs on every session, it's not worth burdening the client with it. I think the chances of this failing are relatively low, and if it fails, the next session will cause it to retry. @dmanjunath wdyt?

Reply:
yeah i don't think we need retries, at least not now